src/hpr_max_copies.erl

```diff
@@ -0,0 +1,140 @@
+-module(hpr_max_copies).
+
+-export([
+    init/0,
+    check/2,
+    delete/1
+]).
+
+-define(ETS, hpr_max_copies_ets).
+-define(MAX_COPIES, max_copies).
+-define(MAX_TOO_LOW, max_copies_max_too_low).
+-define(CLEANUP, timer:hours(1)).
+-define(CLEANUP_SLEEP, timer:hours(1)).
+
+-spec init() -> ok.
+init() ->
+    ets:new(?ETS, [
+        public,
+        named_table,
+        set,
+        {write_concurrency, true}
+    ]),
+    ok = scheduled_cleanup(?CLEANUP),
```

Comment:
If this crashes for some reason, then the cleanups stop. Should this be a supervised process?

Reply:
This isn't a process. It's called by the supervisor.

Reply:
Still, if the process running the cleanup were to crash for some reason, the cleanups stop running.

Reply:
Oh, indeed. 
I misattributed the original message to the making of the ets table.